docs/web/using_with_react/ssr/ssr.mdx

```diff
@@ -5,25 +5,71 @@ sidebar_label: SSR support
 slug: /web/using_with_react/ssr/ssr
 ---
 
-To use Tolgee with your SSR framework such as Next or Gatsby, you should provide localization data imported as object
-using staticData property of TolgeeProvider component, so in production mode your data will be ready when your app is
-rendered on server, or your static page is built from sources.
+To use Tolgee with your SSR framework such as Next or Gatsby, you can provide localization data imported as object
+using staticData property of TolgeeProvider component, this data will be loaded directly into cache and immediately
+available with first render for SSR. And we also need to know user's locale already on the server and force Tolgee
```

Comment:
I would remove the "And" from the beginning of the sentence.

Reply:
👍